src/UIComponents/ColumnInspector.jsx

```diff
@@ -13,6 +13,10 @@ import {
 } from "../redux";
 import { ColumnTypes, styles } from "../constants.js";
 import { Bar } from "react-chartjs-2";
+import ScatterPlot from "./ScatterPlot";
+import CrossTab from "./CrossTab";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTimes } from "@fortawesome/free-solid-svg-icons";
```

Comment:
@made-line this is what I was talking about regarding icons in #107  - I used a shortcut where the icon only shows up in the context of code studio, but should arguably switch to this format so it shows up in standalone too.

Reply:
Gotcha! Thank you :)